### ISSUE TITLE
window control buttons (close, minimize, maximize) have been moved to the sidebar

### ISSUE THREAD

**Issue by tubbadu** [zen-browser/desktop#3640]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [x] I am not filing an enhancement request.

### What happened?

two updates ago the window control buttons were placed on the top toolbar, where "normal" windows usually have them. then, an update (the one that changed the icon) moved them to the sidebar, as shown in the screenshot: 
![image](https://github.com/user-attachments/assets/db428926-9d1a-4482-9805-01f63a6b597e)

This happens both with compact mode enabled or disabled. I don't have a userChrome.css (I had it but removed it to see if it was the problem, and nothing changed). I'm running Zen browser installed from the zip file on Arch Linux

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

Mozilla Zen 1.0.2-b.1

### What platform are you seeing the problem on?

Linux

**Comment by NyaomiDEV** [comment]
I wonder how you got that because I was searching exactly for how to have those in the sidebar instead! Are you sure you don't have them on the right side in normal windows? (Qt and GTK)

**Comment by tubbadu** [comment]
> I wonder how you got that because I was searching exactly for how to have those in the sidebar instead! Are you sure you don't have them on the right side in normal windows? (Qt and GTK)

I found the solution: in system settings > Look and feel, there's this option:

![Image](https://github.com/user-attachments/assets/2ea1aa12-8ceb-48fd-a20b-ed339bd9dd68)

an update automatically switched me to "single toolbar", so manually setting "multiple toolbars" fixed the problem for me. I don't know if this option is new or if I'm just blind, because I looked for it before posting the issue and didn't find it
I hope this helps you!

**Comment by DasKing-SYSLOV** [comment]
Yeah, it's only a problem for arch Linux, even in multiple toolbars it has this problem. It displays the 3 buttons on my windows installation , really weird tho, because up until last week it worked well. Not a dealbreaker tho, if you use KDE you can use the show desktop to minimize the window, for now.

![Image](https://github.com/user-attachments/assets/f4d13e80-704a-42a6-9db5-e19788d09ce6)

**Comment by mmol67** [comment]
> Yeah, it's only a problem for arch Linux, even in multiple toolbars it has this problem. It displays the 3 buttons on my windows installation , really weird tho, because up until last week it worked well. Not a dealbreaker tho, if you use KDE you can use the show desktop to minimize the window, for now.
> 
> ![Image](https://github.com/user-attachments/assets/f4d13e80-704a-42a6-9db5-e19788d09ce6)

Same thing, only close button, in OpenSuse Tumbleweed, Flatpak installation, no matter multiple or single toolbar used.

**Comment by ameyodas** [comment]
I just switched from X11 to wayland, and the menu buttons disappeared.

**Comment by FynnND** [comment]
I have the same problem on CachyOS (KDE) with the flatpak version of Zen. I didn't have that problem on Fedora KDE 42. Both used Wayland and the flatpak version 

Edit: For anyone that has the same issue with missing maximise / minimise and restore down button on Arch I found a workaround. If you use the AUR version instead of the flatpak version of the zen browser (zen-browser-bin) you get those buttons again. I had the same issue on waterfox on another arch based device

**Comment by chandrayan0417** [comment]
How I can hide the close button
